pkg/apis/batch/v1alpha1/job.go

```diff
@@ -193,7 +193,7 @@ type LifecyclePolicy struct {
 	// according to this code.
 	// Note: only one of `Event` or `ExitCode` can be specified.
 	// +optional
-	ExitCode *int32
+	ExitCode *int32 `json:"exitCode,omitempty" protobuf:"bytes,3,opt,name=exitCode"`
```

Comment:
Good catch!

It should be `5`, `3` is used by `Events` :)

Reply:
got it,i have changed this.